### PR TITLE
Voron 0.1 Neopixel Holder 

### PR DESCRIPTION
**Pull request by nikolasbabetas**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
My mod ("Neopixel_Holder"), allows Voron 0.1 users to easily attach Neopixel/WS2812B Led Strip to top extrusion of their printers.
#### How was it tested? 
The parts were printed and have been used on my printer successfully for over a month.
#### Any background context you want to provide?
I could not find any neopixel holders for the V0.1 so I designed my own in Fusion 360.
#### Screenshots (if appropriate)
-
#### Further notes
-

### CODE REVIEW

**Comment by cbc02009** [comment]
Can you please update the printer_mods/readme.md file with your mod?

**Comment by nikolasbabetas** [comment]
I just did!

File: printer_mods/README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ like so:
 |Nashuar|[V0 Dragon Toolhead M2.5x8](./Nashuar/V0_Dragon_ToolheadStockScrews) |This toolhead uses the stock M2.5 screw that came with the dragon hotend |:heavy_check_mark:|:x:|:x:|:x:|
 |natewalck |[V1.8 Jigs](./natewalck/v1.8_Jigs) |Jigs for assembling V1.8 |:x: |:heavy_check_mark: |:x: |:x:|
 | nhchiu | [V0.1 Movable Pulley Z Drive](./nhchiu/V0.1_Movable_Pulley_Z) | A movable pulley system on the Z axis with 1:8 total speed ratio for V0.1 | :heavy_check_mark: | :x: | :x: | :x: |
+| nikolasbabetas | [Neopixel Holder ](./nikolasbabetas/Neopixel_Holder) | A holder to attach Neopixel/WS2812B Led Strips to the V0.1 | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (cbc02009) [inline printer_mods/README.md:213]: Can you add an :x: for the trident column?

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 1 above)
Comment (nikolasbabetas) [reply printer_mods/README.md]: Yes, I added it now

**Comment by nikolasbabetas** [comment]
I have made all the changes requested.

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**